extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -335,6 +335,14 @@ private void registerExtraScalarsInSchema(List<ExtraScalar> extraScalars) {
             switch (extraScalar) {
                 case UUID:
                     Scalars.addUuid();
+                case URL:
+                case LOCALE:
+                case COUNTRY_CODE:
+                case CURRENCY:
+                    Scalars.registerCustomScalarInSchema(
```

Comment:
This method will have to be added in smallrye right? Could you switch UUID to use it too, while you're at it?

Reply:
Method is already there. Sure, I can tweak UUID to use the same